modules/sdk-coin-trx/src/trx.ts

```diff
@@ -250,6 +250,84 @@ export class Trx extends BaseCoin {
   }
 
   async verifyTransaction(params: VerifyTransactionOptions): Promise<boolean> {
+    const { txParams, txPrebuild } = params;
+
+    if (!txParams) {
+      throw new Error('missing txParams');
+    }
+
+    if (!txPrebuild) {
+      throw new Error('missing txPrebuild');
+    }
+
+    if (!txPrebuild.txHex) {
+      throw new Error('missing txHex in txPrebuild');
+    }
+
+    const rawTx = txPrebuild.txHex;
+    const txBuilder = getBuilder(this.getChain()).from(rawTx);
+    const tx = await txBuilder.build();
+    const txJson = tx.toJson();
+
+    try {
+      if (!txJson.raw_data || !txJson.raw_data.contract || txJson.raw_data.contract.length !== 1) {
+        throw new Error('Number of contracts is greater than 1.');
+      }
+
+      const contract = txJson.raw_data.contract[0];
+
+      if (contract.type === 'TransferContract') {
+        return this.validateTransferContract(contract, txParams);
+      } else {
+        return true;
+      }
+    } catch (e) {
+      console.debug('TSS transaction verification failed:', e.message);
+      throw e;
+    }
```

Comment:
no need of try catch.


```suggestion
      if (!txJson.raw_data || !txJson.raw_data.contract || txJson.raw_data.contract.length !== 1) {
        console.debug('TSS transaction verification failed:', e.message);
        throw new Error('Number of contracts is greater than 1.');
      }

      const contract = txJson.raw_data.contract[0];

      if (contract.type === 'TransferContract') {
        return this.validateTransferContract(contract, txParams);
      }
      return true;
    }
```